resolwe_bio/tools/check_bam_source.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+"""Check if the source of RNA ae rRNA or globin RNA."""
+
+import argparse
+import os
+import sys
+
+MAPPINGS_DIR = '/opt/chrom_mappings/globin_rrna/'
+QC_FILES = {
```

Comment:
These are definitely not QC files. I suggest naming this `GLOBIN_RRNA_FILES` and add a comment "These are files that contain names of globin and rRNA sequences"